clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -688,28 +688,26 @@ static Value *emitRangedBuiltin(CodeGenFunction &CGF,
 }
 
 namespace {
-  struct WidthAndSignedness {
-    unsigned Width;
+  struct RangeAndSignedness {
+    unsigned Range;
```

Comment:
I think it would be good to upstream this NFC rename to reduce the diff.

Reply:
Hm, I was confused and our getIntWidth in fact gives the range, which I guess makes sense in C land where this is. Our getIntRange is just a wrapper for getIntWidth, and I suppose is a stalled effort to distinguish the two, which likely needs a hell of a lot more work and commitment from upstream to adopt. So probably this renaming should just be dropped?

Reply:
Still worth upstreaming https://github.com/llvm/llvm-project/pull/101765/files as a minor thing though